mappings/net/minecraft/block/PinkPetalsBlock.mapping

```diff
@@ -2,4 +2,6 @@ CLASS net/minecraft/unmapped/C_onsqqphm net/minecraft/block/PinkPetalsBlock
 	FIELD f_crwyarkl MAX_PETAL_AMOUNT I
 	FIELD f_hvhlkcdp FACING Lnet/minecraft/unmapped/C_tcujuvjb;
 	FIELD f_ierdplen AMOUNT Lnet/minecraft/unmapped/C_vltzvhxi;
+	FIELD f_jprjgbfu FACING_AND_AMOUNT_TO_SHAPE Ljava/util/function/BiFunction;
```

Comment:
```suggestion
	FIELD f_jprjgbfu SHAPE_BY_PROPERTIES
 Ljava/util/function/BiFunction;
```
This is what Mojang calls it, and it follows the `valuesByKeys` format from [CONVENTIONS.md#collections](https://github.com/QuiltMC/quilt-mappings/blob/1.20.2-pre4/CONVENTIONS.md#collections). I also prefer the conciseness of `PROPERTIES` to `FACING_AND_AMOUNT`